content/en/gitops.md

```diff
@@ -1,31 +1,25 @@
 ---
 title: GitOps
-status: Feedback Appreciated
+status: Completed
 category: Concept
-tags: ["", "", ""]
+tags: ["methodology"]
 ---
 
 ## What it is
 
-GitOps is a set of best practices based on [shared principles](https://opengitops.dev/), 
-applied to a workflow that depends on software agents that 
-enable automation to reconcile a declared system state or configuration in a git repository.
-These software agents and practices are used to execute a cohesive workflow that 
-leverages a source control system like Git as the “single source of truth” and 
-extends this practice to applications, infrastructure, and operational procedures.
+GitOps is a set of best practices based on [shared principles](https://github.com/open-gitops/documents/blob/release-v1.0.0/PRINCIPLES.md), applied to a workflow that depends on software agents to continuously and automatically reconcile an actual running system with its desired state declared in an immutably versioned state store like Git.
+These software agents and practices are used to execute a cohesive workflow and extends this practice to applications, infrastructure, and operational procedures.
 
 ## Problem it addresses
 
-Existing processes for infrastructure configuration management can face challenges 
-such as configuration drift, failed deployments, relying on a system's previous state for success, 
-missing documentation, or unknown development history.
+Existing processes for infrastructure configuration management – including scripting and tightly coupled CI/CD – are often error prone and can face challenges such as configuration drift, failed deployments, relying on a system's previous state for success, missing documentation, or unknown development history.
+Event driven CD does not address drift that may happen independently of CI.
+Giving external automation tools credentials to your production systems widens the attack footprint.
 Adopting a GitOps workflow can help alleviate these issues, among several others.
 
 ## How it helps
 
-GitOps is a paradigm that can be applied to a workflow 
-to help manage an application and cloud system infrastructure. 
-It enables organizations several advantages 
-such as better coordination, transparency, stability, and reliability of a system.
-Operating in a close loop ensures the current live state of a system matches 
-against the desired target state, specified in the git repository.
+The GitOps paradigm can be applied to a workflow to help manage applications and cloud system infrastructure. 
+It enables organizations several advantages such as better coordination, transparency, stability, and reliability of a system, and faster mean time to recovery.
```

Comment:
Advantages im comparison to what? I think there are many things that create the mentioned advantages. "Simple" CICD does that as well. IMHO the specific advantage of GitOps in comparison to "simple" CICD should be in focus.